### PR TITLE
Add support for Symfony 6

### PR DESCRIPTION
**Pull request by loic425**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -16,8 +16,13 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: ["7.3", "7.4", "8.0"]
-                symfony: ["4.4.*", "5.1.*"]
+                php: ["7.3", "7.4"]
+                symfony: ["5.3.*", "5.4.0"]
```
Comment (lchrusciel) [inline .github/workflows/build.yml:20]: Why did you removed Sf5.3 & PHP8.0 build?

Comment (loic425) [reply .github/workflows/build.yml:20]: This is below :)

Comment (loic425) [reply .github/workflows/build.yml:20]: OOPs you're right!

Comment (loic425) [reply .github/workflows/build.yml:20]: @lchrusciel Fixed

**Comment by loic425** [comment]
@pamil Hey, great to see you here 👋

**Review by lchrusciel — APPROVED** [review]

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -45,7 +52,7 @@ jobs:
 
             -
                 name: Run Psalm
-                run: vendor/bin/psalm src --no-progress
+                run: vendor/bin/psalm src --no-progress --php-version="${{ matrix.symfony }}"
```
Comment (pamil) [inline .github/workflows/build.yml]:
```suggestion
                run: vendor/bin/psalm src --no-progress --php-version="${{ matrix.php }}"
```

**Review by pamil — CHANGES_REQUESTED** [review]

**Comment by pamil** [comment]
@loic425 thanks for your help, it means a lot!

**Comment by loic425** [comment]
@pamil Could you run one last time please, I ran it on local and that's now ok.

**Comment by lchrusciel** [comment]
@loic425 done

**Review by pamil — APPROVED** [review]

**Comment by pamil** [comment]
Thanks, Loic!

**Pull request merged**